src/analyzers/concept/vehicle-purchase/index.ts

```diff
@@ -46,11 +46,18 @@ return param === true
 
 const USE_IF_ELSE = factory<'function'>`
 Nice.
-That's an \_early return\_.
+That's an \_early return\_ used in the \`${'function'}\` function.
 For the purpose of the Concept that this exercise aims to teach, try solving this
 using an \`else\` statement.
 `('javascript.vehicle-purchase.use_if_else', CommentType.Informative)
 
+const AVOID_DUPLICATE_LITERAL_USAGE = factory`
+TODO: How to inform possible best practice?
```

Comment:
I am not sure how best to inform the student of a better way to not duplicate usage.

Reply:
Let me think about this for a hot minute!

Reply:
I sent @pertrai1 a suggestion for this.